app/models/user/User.scala

```diff
@@ -174,9 +174,10 @@ object UserDAO extends SecuredBaseDAO[User] {
 
   def findOneByEmail(email: String)(implicit ctx: DBAccessContext) = findOne("email", email)
 
-  def findByTeams(teams: List[String], includeAnonymous: Boolean)(implicit ctx: DBAccessContext) = withExceptionCatcher {
+  def findByTeams(teams: List[String], includeAnonymous: Boolean, includeInactive: Boolean = true)(implicit ctx: DBAccessContext) = withExceptionCatcher {
     val anonymousFilter = if(includeAnonymous) Json.obj() else Json.obj("_isAnonymous" -> Json.obj("$ne" -> true))
```

Comment:
Not necessarily related to this PR, but: do we still have anonymous users or did that die with mturk? I only found one in the DB (named MTurk).

Reply:
Good question, I suppose not, then